fastly/connection.py

```diff
@@ -19,6 +20,8 @@ def __init__(self, host='api.fastly.com', secure=True, port=None, root='',
         self.http_conn = None
     
     def request(self, method, path, body=None, headers={}):
+        headers['User-Agent'] = 'fastly-py-v{}'.format(__version__)
+
```

Comment:
I added a `print headers` here, and all tests included our `'User-Agent'` key/value pair.